x500/cert/acme/src/main/java/org/wildfly/security/x500/cert/acme/AcmeClientSpi.java

```diff
@@ -18,65 +18,7 @@
-import static org.wildfly.security.x500.cert.acme.Acme.GET;
-import static org.wildfly.security.x500.cert.acme.Acme.HEAD;
-import static org.wildfly.security.x500.cert.acme.Acme.IDENTIFIER;
-import static org.wildfly.security.x500.cert.acme.Acme.IDENTIFIERS;
-import static org.wildfly.security.x500.cert.acme.Acme.INSTANCE;
-import static org.wildfly.security.x500.cert.acme.Acme.INVALID;
-import static org.wildfly.security.x500.cert.acme.Acme.JOSE_JSON_CONTENT_TYPE;
-import static org.wildfly.security.x500.cert.acme.Acme.JSON_CONTENT_TYPE;
-import static org.wildfly.security.x500.cert.acme.Acme.JWK;
-import static org.wildfly.security.x500.cert.acme.Acme.KID;
-import static org.wildfly.security.x500.cert.acme.Acme.LOCATION;
-import static org.wildfly.security.x500.cert.acme.Acme.META;
-import static org.wildfly.security.x500.cert.acme.Acme.NONCE;
-import static org.wildfly.security.x500.cert.acme.Acme.OLD_KEY;
-import static org.wildfly.security.x500.cert.acme.Acme.ONLY_RETURN_EXISTING;
-import static org.wildfly.security.x500.cert.acme.Acme.PAYLOAD;
-import static org.wildfly.security.x500.cert.acme.Acme.PEM_CERTIFICATE_CHAIN_CONTENT_TYPE;
-import static org.wildfly.security.x500.cert.acme.Acme.PENDING;
-import static org.wildfly.security.x500.cert.acme.Acme.POST;
-import static org.wildfly.security.x500.cert.acme.Acme.PROBLEM_JSON_CONTENT_TYPE;
-import static org.wildfly.security.x500.cert.acme.Acme.PROTECTED;
-import static org.wildfly.security.x500.cert.acme.Acme.RATE_LIMITED;
-import static org.wildfly.security.x500.cert.acme.Acme.REASON;
-import static org.wildfly.security.x500.cert.acme.Acme.REPLAY_NONCE;
-import static org.wildfly.security.x500.cert.acme.Acme.RETRY_AFTER;
-import static org.wildfly.security.x500.cert.acme.Acme.STATUS;
-import static org.wildfly.security.x500.cert.acme.Acme.SUBPROBLEMS;
-import static org.wildfly.security.x500.cert.acme.Acme.TERMS_OF_SERVICE;
-import static org.wildfly.security.x500.cert.acme.Acme.TOKEN;
-import static org.wildfly.security.x500.cert.acme.Acme.URL;
-import static org.wildfly.security.x500.cert.acme.Acme.SIGNATURE;
-import static org.wildfly.security.x500.cert.acme.Acme.TERMS_OF_SERVICE_AGREED;
-import static org.wildfly.security.x500.cert.acme.Acme.TITLE;
-import static org.wildfly.security.x500.cert.acme.Acme.TYPE;
-import static org.wildfly.security.x500.cert.acme.Acme.USER_ACTION_REQUIRED;
-import static org.wildfly.security.x500.cert.acme.Acme.USER_AGENT;
-import static org.wildfly.security.x500.cert.acme.Acme.VALID;
-import static org.wildfly.security.x500.cert.acme.Acme.VALUE;
-import static org.wildfly.security.x500.cert.acme.Acme.WEBSITE;
-import static org.wildfly.security.x500.cert.acme.Acme.base64UrlEncode;
-import static org.wildfly.security.x500.cert.acme.Acme.getAlgHeaderFromSignatureAlgorithm;
-import static org.wildfly.security.x500.cert.acme.Acme.getJwk;
+import static org.wildfly.security.x500.cert.acme.Acme.*;
```

Comment:
Just a small style comment, we tend to avoid using any form of wildcard import :)

Reply:
Fixed!